src/Tizen.NUI.WindowSystem/src/public/QuickPanelClient.cs

```diff
@@ -260,6 +294,55 @@ public QuickPanelClient(TizenShell tzShell, Window win, Types type)
             if (width > height) _screenOrientation = Window.WindowOrientation.Landscape;
         }
 
+        /// <summary>
+        /// Creates a new Quickpanel Client handle.
+        /// </summary>
+        /// <param name="tzShell">The TizenShell instance.</param>
+        /// <param name="win">The window provider for the quickpanel service.</param>
+        /// <param name="type">The type of quickpanel service.</param>
+        /// <exception cref="Tizen.Applications.Exceptions.OutOfMemoryException">Thrown when the memory is not enough to allocate.</exception>
+        /// <exception cref="ArgumentException">Thrown when failed of invalid argument.</exception>
+        /// <exception cref="ArgumentNullException">Thrown when a argument is null.</exception>
+        /// <since_tizen> 12 </since_tizen>
+        public QuickPanelClient(TizenShell tzShell, IWindowProvider win, Types type)
+        {
+            int width = 0, height = 0;
+            if (tzShell == null)
+            {
+                throw new ArgumentNullException((string)"tzShell");
+            }
+            if (tzShell.GetNativeHandle() == IntPtr.Zero)
+            {
+                throw new ArgumentException("tzShell is not initialized.");
+            }
+            if (win == null)
+            {
+                throw new ArgumentNullException((string)"win");
+            }
+
+            _tzsh = tzShell;
+            if (win is NUI.Window nuiWin)
+            {
+                _tzshWin = nuiWin.GetNativeId();
+            }
+            else
+            {
+                throw new ArgumentNullException("win is not NUI Window, so Window Native ID should be set in native level.");
+                //Todo: There is no API in IWindowProvider to get the window ID, so native code is required to set the ID with the window handle.
+            }
```

Comment:
WindowHandle로 부터 NativeId를 얻어와서 설정하면 될것 같습니다. 
```suggestion
            _tzsh = tzShell;
            _tzshWin = ecore_wl2_window_id_get(win.NativeHandle);
```
``` c#
            
            [DllImport("libecore_wl2.so.1")]
            public static extern int ecore_wl2_window_id_get(IntPtr window);

```

Reply:
리뷰 감사드립니다.
가이드 주신데로 수정하였습니다.